test/temporary-financial-reports/cashFlow.js

```diff
@@ -0,0 +1,171 @@
+/* global expect, chai, agent */
+const expect = require('chai').expect;
+const _ = require('lodash');
+
+describe('Cash Flow Report', () => {
+
+  const reportURL = '/reports/finance/cashflow';
+
+  const year2016 = { 
+    account_id: '190',
+    cashbox: '4',
+    dateFrom: '2016-01-01',
+    dateTo: '2016-12-31',
+    lang: 'fr',
+    renderer: 'json',
+    reportId: '1',
+    saveReport: '0',
+    weekly: '0' 
+  };
+
+  const may2015 = { 
+    account_id: '190',
+    cashbox: '4',
+    dateFrom: '2015-05-01',
+    dateTo: '2015-05-31',
+    lang: 'fr',
+    renderer: 'json',
+    reportId: '1',
+    saveReport: '0',
+    weekly: '1' 
+  };
+
+  var periodicBalance = {
+    jan : 1131.42,
+    feb : 1061.45,
+    mar : 991.54,
+    apr : 924.43,
+    may : 855.9599999999999,
```

Comment:
This is not a valid currency amount that we can display - if the server returns this, it means that we are doing some mathematics in JS.  We should make sure all the logic is in SQL.

I'm not sure if this should be blocking - @sfount, what do you think?